packages/components/psammead-podcast-promo/src/components/card-call-to-action.jsx

```diff
@@ -14,10 +14,10 @@ const CallToActionText = styled.span`
   color: ${C_METAL};
 `;
 const CardCallToAction = ({ children, ...props }) => (
-  <span aria-hidden="true">
-    <SeriesStackIcon />
+  <p>
+    <SeriesStackIcon focusable="false" />
     <CallToActionText {...props}>{children}</CallToActionText>
-  </span>
+  </p>
```

Comment:
Do we still need the wrapping element now that this is no longer aria-hidden?

Reply:
Yea good point - I think I misunderstood which element should be a `p` here.

I've changed the `CallToAction` text to be a `p` and wrapped the elements in a React Fragment rather than a span. Screen reader seems to read the contents of the `CallToActionText` correctly.